geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/CONST_CHANGELOG.txt

```diff
@@ -8,6 +8,189 @@ Version 2.7.0
 Information to know before starting the upgrade
 ===============================================
 
+The Upgrade to the Web Components started, the authentication and the LIDAR profile are migrated.
+In the desktop alt we introduce a canvas component who manage the application layout and plug the
+components throw slots.
+In ngeo the sources files of gmf are moved in ngeo src folder.
```

Comment:
```suggestion
In ngeo, the sources files of GMF are moved in ngeo src folder.
```
:-)

Reply:
It's the folder => changed to `contribs/gmf/src`

Reply:
I know, I was just doing some syntax checking, but maybe add it to the sentence if it's better to understand.